KomaMRIBase/src/datatypes/sequence/RF.jl

```diff
@@ -106,6 +106,7 @@ end
 # Properties
 size(r::RF, i::Int64) = 1 #To fix [r;r;;] concatenation of Julia 1.7.3
 *(α::Complex{T}, x::RF) where {T<:Real} = RF(α*x.A,x.T,x.Δf,x.delay)
+*(α::Real, x::RF) = RF(α*x.A,x.T,x.Δf,x.delay)
```

Comment:
Why not `flipangle(r::RF)` and `set_flipangle(r::RF, angle)` here? As analogs to the area functions.